test/idp/src/main/kotlin/org/codice/compliance/web/sso/PostLoginTest.kt

```diff
@@ -123,8 +124,30 @@ class PostLoginTest : StringSpec() {
             SingleSignOnProfileVerifier(responseDom, acsUrl[HTTP_POST]).verify()
         }
 
+        "POST AuthnRequest With Relay State Greater Than 80 Bytes Test" {
+            Log.debugWithSupplier { "POST AuthnRequest With Relay State Greater Than 80 Bytes Test" }
+            val encodedRequest = Encoder.encodePostMessage(
+                    createValidAuthnRequest(), TestCommon.RELAY_STATE_GREATER_THAN_80_BYTES)
+            val response = given()
+                    .urlEncodingEnabled(false)
+                    .body(encodedRequest)
+                    .contentType("application/x-www-form-urlencoded")
+                    .log()
+                    .ifValidationFails()
+                    .`when`()
+                    .post(Common.getSingleSignOnLocation(POST_BINDING))
+
+            val idpResponse = TestCommon.parseErrorResponse(response)
+
+            idpResponse.bindingVerifier().verifyError(TestCommon.IDP_ERROR_RESPONSE_REMINDER_MESSAGE)
+
+            val responseDom = idpResponse.responseDom
+
+            CoreVerifier(responseDom).verifyErrorStatusCode(SAMLBindings_3_5_3_a, TestCommon.REQUESTER)
```

Comment:
✏️ Unfortunately, I can't just read this line and know what it's doing. It'd be nice to either 1) have a convenience function like `verifyRequesterStatusCode(samlErrorCode: SAMLSpecRefMessage)` or 2) be explicit with the name of the parameters `verifyErrorStatusCode(samlErrorCode = <SAMLBindingsblah>, expected = <REQUESTER>`.

Reply:
Went with numero dos